tasks.c

```diff
@@ -3380,7 +3454,13 @@ void vTaskSuspendAll( void )
         }
         else
         {
-            xReturn = xNextTaskUnblockTime - xTickCount;
+            /* Declare xLocalNextTaskUnblockTime/xLocalTickCount here to confort MISRA C 2012 Rule 13.2 - 

```

Comment:
Suggest we split it into the following instruction instead of using local variables
```
xReturn = xNextTaskUnblockTime;
xReturn = xReturn - xTickCount;
```

Reply:
Suppress because it's scanned by advisory checker.